test/unit_tests/modules/util/standard-type-spec.js

```diff
@@ -7,5 +7,15 @@ var expect = chai.expect;
 var standardType = require( BASE_JS_PATH + 'modules/util/standard-type' );
 
 describe( 'standard-type', function() {
-  // TODO: Implement tests.
+  it( 'should include a standard JS data hook', function() {
+    expect( standardType.JS_HOOK ).to.equal( 'data-js-hook' );
+  } );
+
+  it( 'should include an filler function', function() {
+    expect( standardType.noopFunct() ).to.be.undefined;
```

Comment:
ok, I would change the language to read `should include a non operational function`